lib/dal/src/workspace_snapshot/graph.rs

```diff
@@ -1038,7 +1040,7 @@ impl WorkspaceSnapshotGraph {
         );
         let filename_no_extension = format!("{}-{}", Ulid::new().to_string(), suffix);
 
-        let home_str = std::env::var("HOME").unwrap();
+        let home_str = std::env::var("HOME").expect("could not find home directory via env");
```

Comment:
We allow expects in debug output functions. This is within a function to prints out a "dot" graph for local development only.